theano/gof/op.py

```diff
@@ -152,7 +152,7 @@ def c_lib_dirs(self):
 
     def c_support_code(self):
         """
-        Optional: Return utility code for use by a `Variable` or `Op` to be
+        Optional: Return utility code (a string, or a LIST of strings) for use by a `Variable` or `Op` to be
```

Comment:
Can you check in the documentation, in particular `doc/extending/cop.txt` and `doc/extending/extending_theano_c.txt`, to see if references to `c_support_code` need to be updated?